rasa_core_sdk/endpoint.py

```diff
@@ -69,7 +69,14 @@ def health():
     def webhook():
         """Webhook to retrieve action calls."""
         action_call = request.json
-        response = executor.run(action_call)
+        try:
+            response = executor.run(action_call)
+        except ActionExecutionError as e:
+            logger.error(str(e))
+            result = {"error": str(e), "action_name": e.action_name}
```

Comment:
`"name"` instead of `"action_name"`? not sure which one is more consistent

Reply:
let's keep `action_name` because it is `action_name` in the error